### PR TITLE
Initial Setup

### PR DESCRIPTION
**Pull request by silabs-robin**
This PR merges `core-v-verif` into `40s-dv`.

That is: It merges the `git subtree` of `core-v-verif/cv32e40s/` (from `core-v-verif:cv32e40s/dev`) into `cv32e40s-dv`.

--------

Method:
1. I cloned and checked out `core-v-verif:cv32e40s/dev`.
2. Then I ran `git subtree split --prefix cv32e40s -b sdv_first`.
    * (Note: It took quite some minutes to process the >5000 commits.)
    * (Note: The resulting `sdv_first` branch contains only the content from the `core-v-verif/cv32e40s/` directory and commits that have changed files in that directory.)
3. Finally I did `git merge s-dv/main  --allow-unrelated-histories --no-commit`
    * (Note: The subtree branch from step 2 couldn't be PRed because there was no common ancestor, this fixes that.)

--------

Test status:
* ci_check - All green.
* Formal (s-dv) - Runable.
* Formal (internal) - Runable.
* (Note: Using `CV_CORE=cv32e40s  VERIF_ENV_REPO=git@github.com:silabs-robin/cv32e40s-dv.git  VERIF_ENV_REF=sdv_first_ancestorized  ./bin/clonetb --clone`.)

Just works. No tweaking was needed.

### CODE REVIEW

File: README.md
Comment (silabs-robin) [inline README.md:1]: Note: For a transition period, the method for syncing `core-v-verif:cv32e40s/dev` up to this `s-dv` repo is pretty much the same as I used for creating this initial PR - let `git subtree split` run, then PR the results.

**Comment by silabs-robin** [comment]
@MikeOpenHWGroup, @silabs-krdosvik.

I have 0 permissions on this repo so I can't add you as reviewers, but I would like to ask you both for a review.

**Comment by MikeOpenHWGroup** [comment]
While I did not look at all 499 files, the structure of this PR matches my expectations, so I am approving it.  I assume we will test this before removing the cv32e40s directory from CORE-V-VERIF in any case ("make-before-break" testing).

**Review by MikeOpenHWGroup — APPROVED** [review]

**Review by silabs-krdosvik — APPROVED** [review]
Agree with Mike, it looks like excepted (but have not looked through them all either). 
Considering that ci check passes, formal compiles and I agree with the method of extracting, I think this is good.

**Comment by silabs-robin** [comment]
> I did not look at all 499 files

Oh really? :)


> I assume we will test this before removing the cv32e40s directory

Definitely.
I forecast the S directory might live for at least a month (guessing).

--------

> I agree with the method of extracting

Thanks. That's one of the most important things I wanted reviewed.

--------

Btw, I don't have any permissions to merge, but since you both have approved I think it is time.

**Pull request merged**